.github/workflows/deploy_prod.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy to production
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
quick question: what value is semver providing here? I'm not clear when I would do anything other than increment MAJOR

Reply:
Yeah, there isn't really a need for semver specifically. We could just use major versions, or follow a date-based format.